webcompat/helpers.py

```diff
@@ -85,10 +85,16 @@ def md5_checksum(file_path):
 
 
 def get_str_value(val):
```

Comment:
This was created by #2395. And probably the name was not great at the start, but now it seems to have drifted a bit. We should probably change that to reflect the new semantics

And this is used by 
https://github.com/webcompat/webcompat.com/blob/9187af92432fc4046c9dcca39be5b6f009fabaea/webcompat/form.py#L296-L309

Reply:
What I'm not sure is that are we trying to do twice the same thing at two different places. It looks very similar and confusing. 

Maybe our tests were not complete at the start.

`get_details` is being tested here

https://github.com/webcompat/webcompat.com/blob/9187af92432fc4046c9dcca39be5b6f009fabaea/tests/unit/test_form.py#L220-L227

Reply:
Yeah, it is a bit confusing. We're essentially going one more level deep so we handle a value that's more complex than a simple string.